addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -219,16 +188,15 @@ widgetsMedia.ImageWidget.include({
      * @private
      */
     _onUnsplashImgClick: function (ev) {
-        var imgid = $(ev.currentTarget).data('imgid');
-        var url = $(ev.currentTarget).data('url');
-        var downloadURL = $(ev.currentTarget).data('download-url');
+        const {imgid, url, downloadUrl} = ev.currentTarget.dataset;
         if (!this.options.multiImages) {
             this._unsplash.selectedImages = {};
+            this.selectedAttachments = [];
         }
         if (imgid in this._unsplash.selectedImages) {
             delete this._unsplash.selectedImages[imgid];
         } else {
-            this._unsplash.selectedImages[imgid] = {url: url, download_url: downloadURL};
```

Comment:
I would prefer no modification of line for nothing, especially since `downloadURL` follows my convention :)

Reply:
This is to allow the destructuring assignment on line 191: `const {imgid, url, downloadUrl} = ev.currentTarget.dataset` so it's not for absolutely no reason, I found it clearer than renaming in the assignment like this:  `const {imgid, url, downloadURL: downloadUrl} = ev.currentTarget.dataset`, but all in all neither of theses changes are really needed, they were more of a "refactoring as I go" kinda thing, would you like me to just keep the original code in both places?

Reply:
No ok, we can keep that :+1: